documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+      1. Submitting the Delegate Report to the WCA Regulations Committee and the WCA Quality Assurance Committee regarding adherence to WCA Regulations and quality standards during the competition, the overall course of the competition, and any incidents that happened. The report must be submitted within one week of the end of the competition.
+      2. Submitting the competition results to the WCA Results Team within one week of the end of the competition.
+         1. Results should be double-checked and free of errors before submission.
+         2. All scramble sequences used during a competition must be sent with the results of the competition.
+            1. All scramble sequences must be labeled with the events, rounds, and groups for which they were used. If there are multiple groups on a round, it is not necessary to identify which competitor was in which group.
+         3. Physical records (e.g. score sheets, written solutions from 3x3x3 Fewest Moves) must be retained for a minimum of 90 days. Digital records (e.g. scramble sequences, results) must be retained for 7 years.
+      3. Sending corrections of competition results to the WCA Results Team.
+      4. Advising other competition officials when necessary.
+      5. Deciding about disqualifying competitors during the competition.
+      6. Generating, providing, and maintaining the security of the scramble sequences ([WCA Regulation 4b2)](wca{regulations/#4b2})).
+      7. Making a copy of the WCA Regulations available at the competition.
+   2. The WCA Delegate may delegate some of their responsibilities to organizers or other officials.
+
+If WCA Regulations are not correctly observed during a competition, results may be modified at the discretion of the WCA Delegate, WCA Regulations Committee, WCA Results Team, WCA Disciplinary Committee, or the WCA Board.
+
+### Announcement Criteria
+The WCA Delegate must submit the following information when requesting approval of a competition:
+
+1. Competition name, nickname, and ID
+   1. The name must not be too general. It should be related in some way to the competition itself, such as:
+      1. The location of the competition
+      2. The competition venue
+      3. The season or date of the competition
+      4. The events of the competition
+      5. The convention the competition is a part of
+      6. A local cultural reference
+      7. A sponsor of the competition
+   2. Capitalization of the competition name must be appropriate. The first letter of each word in the competition’s name should be capitalized and subsequent letters within words should be lowercase, except articles and prepositions, which should be always lowercase. Acronyms should be capitalized.
+   3. The name must end with the year of the last day of the competition.
+   4. The competition name must be in Roman-readable letters and may only contain characters from A-Z, a-z, and 0-9, with or without accents. Only '.:-& are permitted as special characters.
```

Comment:
```suggestion
   4. The competition name must be in Roman-readable letters and may only contain characters similar to A-Z, a-z, and 0-9. Only '.:-& are permitted as special characters.
```
otherwise even with the Roman-readable change, this would read that ł is not allowed.

Reply:
I would prefer to keep the A-z, a-z, 0-9 and add "or similar characters" after, then. The suggestion could be interpreted that the original letters A-Z are not permitted--only letters similar to them. (Yes, I know it's a ridiculous interpretation.)

Reply:
```suggestion
   4. The competition name must be in Roman-readable letters and may only contain characters from A-Z, a-z, 0-9, or similar characters. Only '.:-& are permitted as special characters.
```